templates/exadata-limits/README.md

```diff
@@ -0,0 +1,44 @@
+# exadata-limits
+Terraform Template to get info for exadata limits raising.
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| [azurerm](https://registry.terraform.io/providers/hashicorp/azurerm/latest) | n/a |
+
+
+## Modules
+| Name                                                                  |
+|-----------------------------------------------------------------------|
+| [azure-region-zone](../../modules/azure-oci-zone-mapping) |
+
+
+## Inputs Variables 
+
+| VARIABLE                                   |                              DESCRIPTION                               | Type | REQUIRED | DEFAULT_VALUE |                    SAMPLE VALUE |
+|:-------------------------------------------|:----------------------------------------------------------------------:|:--------:|---------:|--------------:|--------------------------------:|
+| location                                   | Specifies the Azure location where would like to get limits increased. | `string` |      n/a |           yes |        e.g. "East US", "eastus" |
+| zones                                      |       The Azure logicalZones would like to get limits increased.       | `string` |       "" |            no |                      e.g. "1,2" |
+| az_far_child_site                          |                       The Azure far child site.                        | `string` |    "N/A" |            no |     e.g. "IAD-52", "IAD-53", "FRA-16" |
+
+
+### Setting param value 
+The following input tfvars *must* be set
+
+Either as `terraform.tfvars` file in same directory
+```
+location="<Location Name e.g. West US or westus>"
```

Comment:
Can we provide a list of all regions/location id for ODB@A

Reply:
I didn't  provide a list of region/locations, it is becasue:
Also when location is wrong, e.g. a typo UK Sout and run terraform apply/plan, it will show the error with valid locations: 
```
 Error: "uksout" was not found in the list of supported Azure Locations: "australiacentral,australiacentral2,australiaeast,australiasoutheast,brazilsouth,brazilsoutheast,
```
When a customer requires limits, customer should know the location. Also, for a different subscription I believe regions could be different.